spot-client/src/common/utils/meeting.test.js

```diff
@@ -0,0 +1,28 @@
+import { findWhitelistedMeetingUrl } from './meeting';
+describe('meeting utils', () => {
+    describe('findWhitelistedMeetingUrl', () => {
```

Comment:
any tests that would cover the exclude pattern?

Reply:
Do you know why the exclude pattern even exists? I can't remember.

Reply:
The jitsi dial in info page...that's why.